client/state/sites/hooks/use-site-options.ts

```diff
@@ -0,0 +1,29 @@
+import { snakeToCamelCase } from '@automattic/js-utils';
+import { useSelector } from 'react-redux';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import getSiteOptions from '../selectors/get-site-options';
+
+type SiteOptions = { [ key: string ]: any };
+
+/**
+ * Get the site options
+ *
+ * @param optionNames options you want to query
+ * @returns {object} site options and the properties are listed in the given option names and converted to camel case
+ */
+const useSiteOptions = ( optionNames: string[] = [] ): SiteOptions => {
```

Comment:
Making the `optionNames` param optional sort of implies you'll get something sensible by default. But in fact the hook will return nothing. I think `optionNames` should instead be either:
- not optional, or
- if omitted then the hook returns all site options.

Reply:
I think we can omit it and return all site options 👍